docs/products/kafka/kafka-connect/howto/debezium-source-connector-mongodb.rst

```diff
@@ -0,0 +1,100 @@
+Create a Debezium source connector for MongoDB
+==================================================
+
+The Debezium source connector for MongoDB tracks database changes using a MongoDB replica set or shared cluster, and writes them to an Apache Kafka® topic in a standard format where they can be transformed and read by multiple consumers.
```

Comment:
Should `using` be `in` (that is, changes in the MongoDB entity)? This might be me not understanding terminology, though.